### ISSUE TITLE
Delete cache on logOut msal-node electronTestApp

### ISSUE THREAD

**Issue by Ladvace** [AzureAD/microsoft-authentication-library-for-js#2729]
I'm working on your sample  [ElectronTestApp](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-node-samples/standalone-samples/ElectronTestApp and the log out doesn't work poperly, when you log out it actually doesn't delete the cache and I can still do a silent login after logout, instead I should be redirect to the login page
).

It seems like  `removeAccount` doesn't delete the account. is there anyway to delete the cache and be redirected to the login page?
```
this.clientApplication
          .getTokenCache()
           .removeAccount(this.account);
```

## Library
- [x] `@azure/msal-node@1.0.0-beta.2`

## Framework
- [x] electron

**Comment by hectormmg** [comment]
Hey @Ladvace, my first guess is there's a problem with the sample's cache plugin, let me take a look and I'll get back to you!

**Comment by hectormmg** [comment]
Turns out there was a bug in the AuthProvider class where the `loginSilent`method returned the cached account but didn't set it under `this.account`, so by the time `logout` was clicked, this.account was null and `removeAccount` wasn't being called. Thanks for catching this bug @Ladvace. Should work fine after merging the fix in #2830.

**Comment by Ladvace** [comment]
Thanks to you for fixing that
